src/logic/safe/transactions/gasNew.ts

```diff
@@ -80,16 +87,23 @@ export const estimateTxGasCosts = async (safeAddress, to, data, tx?: any, preApp
   }
 }
 
-export const estimateSafeTxGas = async (safe, safeAddress, data, to, valueInWei, operation) => {
+export const estimateSafeTxGas = async (
+  safe: GnosisSafe | undefined,
```

Comment:
`safe?: GnosisSafe`?

Reply:
a required parameter cannot be followed by an optional parameter